subxt/src/tx/tx_progress.rs

```diff
@@ -172,7 +172,7 @@ impl<T: Config, C: Clone> Stream for TxProgress<T, C> {
                 //
                 // Even though `Dropped`/`Invalid`/`Usurped` transactions might make it into a block eventually,
                 // the server considers them final and closes the connection, when they are encountered.
-                // curently there is no way of telling if that happens, because the server ends the stream before.
+                // In those cases you have to re-subscribe to the extrinsic/create a new TxProgress repeatedly until the extrinsic is finalized.
```

Comment:
```suggestion
    /// In those cases the stream is closed however, so you currently have no way to find 
    /// out if they finally made it into a block or not.
```

Just to revert this since turns out you can't re-subscribe :)